scripts/test-ci-all.sh

```diff
@@ -25,41 +25,53 @@ cargo build ${CARGO_PROFILE:+--profile ${CARGO_PROFILE}} --all --all-targets
 
 function cli_test_reconnect() {
   set -eo pipefail # pipefail must be set manually again
+  trap 'echo "## FAILED: ${FUNCNAME[0]}"' ERR 
```

Comment:
Unfortunately, I don't think I can re-use anything here, as it seems that each of these gets executed "from scratch" by gnu parallel so all pre-existing bash options are gone, and there's not enough redundancy yet, to make me want to spend more time investigating best way to reuse (maybe common exported function will work, or maybe I need a file).